modules/update-availability-faq.adoc

```diff
@@ -0,0 +1,67 @@
+// Module included in the following assemblies:
+//
+// * updating/understanding-openshift-updates.adoc
+
+:_content-type: CONCEPT
+[id="update-availability_{context}"]
+= Common questions about update availability
+
+There are several factors that affect if and when an update is made available to an {product-title} cluster. The following list provides common questions regarding the availability of an update:
+
+[id="channel-differences_{context}"]
+*What are the differences between each of the update channels?*
+
+* A new release is initially added to the `candidate` channel.
+
+* After successful final testing, a release on the `candidate` channel is promoted to the `fast` channel, an errata is published, and the release is now fully supported.
+
+* After a delay, a release on the `fast` channel is finally promoted to the `stable` channel. This delay represents the only difference between the `fast` and `stable` channels.
++
+[NOTE]
+====
+For the latest z-stream releases, this delay may generally be a week or two. However, the delay for initial updates to the latest minor version may take much longer, generally 45-90 days.
+====
+
+* Releases promoted to the `stable` channel are simultaneously promoted to the `eus` channel.
+The primary purpose of the `eus` channel is to serve as a convenience for clusters performing an EUS-to-EUS update.
+
+[id="channel-safety_{context}"]
+*Is a release on the `stable` channel safer or more supported than a release on the `fast` channel?*
+
+* If a regression is identified for a release on a `fast` channel, it will be resolved and managed to the same extent as if that regression was identified for a release on the `stable` channel.
+
+* The only difference between releases on the `fast` and `stable` channels is that a release only appears on the `stable` channel after it has been on the `fast` channel for some time, which provides more time for new update risks to be discovered.
+
+** This delay is used to further test and gather data about a release that is on the `fast` channel, but otherwise there is no difference between releases on each of the channels.
```

Comment:
Can we replace "to further test and gather data" with "to allow for new update risks to be discovered" or something similar that drops "test" and focus on passive monitoring?  Basically the same point discussed [here][1], but applied to this line.

[1]: https://github.com/openshift/openshift-docs/pull/61657#discussion_r1242629259